includes/class-monk-links.php

```diff
@@ -284,14 +284,24 @@ public function monk_add_language_home_permalink( $link, $path ) {
 			return $link;
 		}
 
-		$url_language  = get_query_var( 'lang' );
-		$language      = ( empty( $url_language ) ) ? $this->site_language : $url_language;
+		$url_language         = get_query_var( 'lang' );
+		$language             = ( empty( $url_language ) ) ? $this->site_language : $url_language;
+		$default_language_url = get_option( 'monk_default_language_url', false );
 
-		if ( $language && '/' === $path ) {
+		if ( '/' === $path || '' === $path ) {
```

Comment:
This function must accept any $path value.
Before return `$link` pass it in `trailingslashit` function.